recipes/blast/build.sh

```diff
@@ -18,7 +18,7 @@ elif [ "$(expr substr $(uname -s) 1 5)" == "Linux" ]; then
     cd $SRC_DIR/c++
     ./configure --prefix=$PREFIX
 
-    make -j4
+    make -j2
```

Comment:
A plain `-j` would achieve the same I think. However, Travis provides 1.5 cores. I am not sure how this is reported via nproc.